recipe/conda_build_config.yaml

```diff
@@ -365,6 +365,16 @@ cairo:
   - 1.16
 cfitsio:
   - 3.470
+coin-or-cbc:
+  - 2.10.5
+coin-or-cgl:
+  - 0.60.3
+coin-or-clp:
+  - 1.17.4
```

Comment:
The current version is 1.17.6 but that has some bugs and at least `or-tools` has a pin to 1.17.4, not sure how to handle this.